open-api/rest-catalog-open-api.py

```diff
@@ -1112,6 +1112,7 @@ class LoadTableResult(BaseModel):
      - `s3.secret-access-key`: secret for credentials that provide access to data in S3
      - `s3.session-token`: if present, this value should be used for as the session token
      - `s3.remote-signing-enabled`: if `true` remote signing should be performed as described in the `s3-signer-open-api.yaml` specification
+     - `expires-at-ms`: if present, specifies timestamp in milliseconds when credentials would expire
```

Comment:
is this supposed to be AWS specific?

Reply:
@munendrasn I don't see anything in the code related to this property. Are you planning to add this property to the current implementation so that it's actually used?

Reply:
yes, once spec is updated.. ie, property name is aligned... planning to add this property. 
This would help any REST spec impl to have custom refresh mechanism until REST client supports refresh and eventually support refresh mechanism for table credentials too like catalog auth refresh.